src/utils/proposals.js

```diff
@@ -0,0 +1,366 @@
+/**
```

Comment:
Because the main class being manipulated by this util is the `PartialTxProposal` and its properties.

I feel renaming it to `atomic-swap` on the file and/or its properties and methods would just increase the name size to two words without bringing more readability or context. On the headless, for example, we have an `atomic-swap` folder, but the name of the controller is `tx-proposal-controller`.

What do you think about this?

Reply:
The lib class creates a proposal (a generic one) but the Desktop Wallet feature is an atomic swap. The controller name is proposal because it's the atomic swap proposal (the controller inside the atomic swap folder).

I think for anyone trying to find files that interact with the atomic swap screens in the desktop wallet would be better to see atomic swap and not a proposal.

We will probably have proposals in the future for sending tokens in a regular transaction.

Reply:
Refactored both the utils and sagas files from `proposals` to `atomicSwap` on [88e3072](https://github.com/HathorNetwork/hathor-wallet/pull/361/commits/88e3072c7a96d43983ff0093e560781f8253db82)